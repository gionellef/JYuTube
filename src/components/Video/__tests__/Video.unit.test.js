import React from 'react';

import { shallow } from 'enzyme';

import { Video } from '../Video';

describe('Video', () => {
  test('renders Video on Watch without prop.id', () => {
    const wrapper = shallow(
      <Video />
    );

    expect(wrapper).toMatchSnapshot();
  });

  test('renders Video on Watch with prop.id', () => {
    const wrapper = shallow(
      <Video id='P5ktQh3sWZk'/>
    );

    expect(wrapper).toMatchSnapshot();
  });
});