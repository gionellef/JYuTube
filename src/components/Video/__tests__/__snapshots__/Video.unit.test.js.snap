// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video renders Video on Watch with prop.id 1`] = `
<div
  className="video-container"
>
  <div
    className="video"
  >
    <iframe
      allow="autoplay; encrypted-media"
      allowFullScreen={true}
      className="video-player"
      frameBorder="0"
      src="https://www.youtube.com/embed/P5ktQh3sWZk"
      title="video"
    />
  </div>
</div>
`;

exports[`Video renders Video on Watch without prop.id 1`] = `""`;
